andela/AndelaEatsUI
# #160993247 User book meal

#### What does this PR do?
This PR allows the user to book a meal

#### Description of Task to be completed?
The menu for the week is displayed for the user to pick. Each day, a user gets to pick among to menu and the user's order is posted

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-user-book-meal-functionality-160993247`
- run these commands `npm run bundle` and `npm run dev` 
- Log in as a `User`, when the `menus` for each day is displayed for you, click on your preferred meal and submit

#### What are the relevant pivotal tracker stories?
- [#160993247](https://www.pivotaltracker.com/n/projects/2178749/stories/160993247)

#### Background Context
Nil

#### Screenshots (If Applicable)
<img width="1036" alt="screen shot 2018-11-23 at 9 16 52 pm" src="https://user-images.githubusercontent.com/21138774/48959385-40a39d80-ef65-11e8-856d-5b239abf8656.png">
<img width="1064" alt="screen shot 2018-11-23 at 9 17 02 pm" src="https://user-images.githubusercontent.com/21138774/48959397-51541380-ef65-11e8-94e0-b4863b33c987.png">
<img width="1063" alt="screen shot 2018-11-23 at 9 17 09 pm" src="https://user-images.githubusercontent.com/21138774/48959401-5e710280-ef65-11e8-9f8a-ce822f7f04ff.png">
